spec/service/default_group_service_spec.rb

```diff
@@ -0,0 +1,121 @@
+require "rails_helper"
+
+RSpec.describe DefaultGroupService do
+  subject(:default_group_service) do
+    described_class.new
+  end
+
+  describe "#create_trial_user_default_group!" do
+    let(:user) { create :user, name: "Batman" }
+    let(:form) { build :form, id: 1 }
+    let(:forms_response) do
+      [form]
+    end
+
+    before do
+      ActiveResource::HttpMock.respond_to do |mock|
+        mock.get "/api/v1/forms?creator_id=#{user.id}", headers, forms_response.to_json, 200
+      end
+    end
+
+    context "when user has a trial role" do
+      it "creates a group if the user does not already have one" do
```

Comment:
"if the user does not already have one" doesn't sound so certain about what it's testing

Using contexts would make it easier to read:

```suggestion
      when "the user does not already have a trial group" do
          it "creates a group" do
```